Fixes #18294: fix display of AK subscription tables.

waldenraines opened the pull request:
http://projects.theforeman.org/issues/18294

jlsherrill commented on engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-subscriptions-list.html:
```diff
@@ -36,8 +36,12 @@
-          <td bst-table-cell colspan="8" ></td>
+          <td class="row-select"></td>
+          <td colspan="8">
+            <a href='/subscriptions/{{ subscription.id }}/info' class="confined-text">
```
this doesn't work for me, as there is no 'subscription' variable here, only subscriptions. 'subscription' is defined below in the ng-repeat, but not here.

jlsherrill commented on engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html:
```diff
@@ -37,8 +37,12 @@
-          <td bst-table-cell colspan="8"></td>
+          <td class="row-select"></td>
+          <td colspan="8">
+            <a href='/subscriptions/{{ subscription.id }}/info' class="confined-text">
```
same issue here

waldenraines commented:
@jlsherrill fixed.

jlsherrill commented:
Just realized the link right below this one here:  https://github.com/waldenraines/katello/blob/92e7185ca1d65728d4ff42da0fdbfd1f3df66215/engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html#L62

is pointing to an invalid url /info doesn't exist, should just be  /subscriptions/id/.  Do you wanna just fix it here? or want me to file an issue?

waldenraines commented:
> Just realized the link right below this one here: https://github.com/waldenraines/katello/blob/92e7185ca1d65728d4ff42da0fdbfd1f3df66215/engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-add-subscriptions.html#L62
> 
> is pointing to an invalid url /info doesn't exist, should just be /subscriptions/id/. Do you wanna just fix it here? or want me to file an issue?

Fixed all places where /info was being used.  Also fixed the links on the content host subscriptions pages that these were based on and which were also going no where.
